extensions/amp-story/1.0/amp-story-interactive-results.js

```diff
@@ -55,25 +60,118 @@ const buildResultsTemplate = (element) => {
 /**
  * Processes the state and returns the condensed results.
  * @param {!Map<string, {option: ?./amp-story-interactive.OptionConfigType, interactiveId: string}>} interactiveState
- * @param {?Array<!./amp-story-interactive.OptionConfigType>} options needed to ensure the first options take precedence on ties
+ * @param {?Array<!./amp-story-interactive.OptionConfigType>} options the attributes on the component
  * @return {InteractiveResultsDef} the results
  */
 const processResults = (interactiveState, options) => {
+  const strategy = decideStrategy(options);
+  if (strategy === 'category') {
+    return processResultsCategory(interactiveState, options);
+  } else if (strategy === 'percentage') {
+    return processResultsPercentage(interactiveState, options);
+  }
+};
+
+/**
+ * Processes the state and returns the condensed results for a category strategy
+ * @param {!Map<string, {option: ?./amp-story-interactive.OptionConfigType, interactiveId: string}>} interactiveState
+ * @param {?Array<!./amp-story-interactive.OptionConfigType>} options the attributes on the component
+ * @return {InteractiveResultsDef} the results
+ * @package
+ */
+export const processResultsCategory = (interactiveState, options) => {
+  const result = {'percentage': null, 'category': null};
```

Comment:
Do you need the quotes to avoid minification errors?

Reply:
I really don't need to specify anything here besides an empty object in reality